services/registration/src/routes/application.ts

```diff
@@ -224,6 +225,26 @@ applicationRouter.route("/:id/actions/submit-application").post(
       throw new BadRequestError("Unable to submit application data. Branch is not currently open.");
     }
 
+    const applicantType = branch.applicationGroup;
+    const autoConfirm = branch.automaticConfirmation;
+
+    if (
+      autoConfirm?.enabled &&
+      (applicantType === "PARTICIPANT" || existingApplication.email in autoConfirm.emails!)
```

Comment:
You shouldn't directly check the applicant type is a participant. In some situations, we still may have automatic confirm enabled for a participant branch, but we'd want to manually check emails/domains. Also, it's best to keep this checking functionality consistent between all the branches or it can be confusing for the end user